elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -51,7 +60,7 @@ $pfe-cta--Color--fallback: #003366;
 
   ::slotted(button),
   ::slotted(input) {
-    @include pfe-reset-button;
+    @extend %reset-button;
```

Comment:
(not really part of this PR but since this file is touched, on lines 53 and 56, background color is defined twice.)

Reply:
The file is touched but because the mixin was changed to an extend.  I'm anxious about removing any styles from pfe-cta because we have been working on fixing it for so long.  I would hate to break something.

Reply:
neither line is scoped to anything new, its just the same definition twice.